catFood.js

```diff
@@ -0,0 +1,32 @@
+var inputCat= document.getElementById("inputCat");
+var getCatFood = new XMLHttpRequest();
+getCatFood.addEventListener("load", printToDomCat);
+getCatFood.open("GET", "catFood.json");
+getCatFood.send();
+
+function printToDomCat () {
+	var catFood= JSON.parse(this.responseText);
+	var string="";
+	// For loops to add to DOM
+	// Brand Name
+	for (i=0; i<catFood.cat_brands.length; i++) {
```

Comment:
It's good practice to put the ```var``` in the declaration of your incrementor variable in the for loop.  Since JavaScript is squishy, it knows what you mean and helpfully treats this as a declaration, but most JS linters will point this out as an error, since you're technically using the variable before it's declared.